### ISSUE TITLE
Make sure M3 users are running VZ emulation and not QEMU

### ISSUE THREAD

**Issue by jandubois** [rancher-sandbox/rancher-desktop#6014]
Current versions of QEMU do not seem to work properly on M3 chips (see https://github.com/lima-vm/lima/issues/1996, https://gitlab.com/qemu-project/qemu/-/issues/1990).

We need to make VZ the default on M3 systems, and also update the settings validator and preferences dialog to prevent M3 users from selecting the QEMU emulation mode.

```
$ system_profiler -json SPHardwareDataType | jq '.SPHardwareDataType[].chip_type'
"Apple M1 Pro"
```

**Comment by breun** [comment]
I ran into this issue, and [switching to VZ emulation](https://docs.rancherdesktop.io/1.11/ui/preferences/virtual-machine/emulation/#vz) indeed worked.

**Comment by jandubois** [comment]
Moving to 1.13; for 1.12 it will just be a release-note. Hopefully the upstream issue is resolved in time for 1.13, so we won't need to add diagnostics, but instead update QEMU to a compatible version.

**Comment by joehorsnell** [comment]
Related #5943
